src/ngraph/op/reshape.hpp

```diff
@@ -117,7 +117,7 @@ namespace ngraph
             {
             public:
                 NGRAPH_API
-                static constexpr NodeTypeInfo type_info{"Reshape", 1};
+                static constexpr NodeTypeInfo type_info{"DynReshape", 1};
```

Comment:
`Reshape:v1` description is `DynReshape` in order to deal with current `get_typeid` implementation (`Reshape:v1` is higher version of `DynReshape:v0`)

Reply:
This is going to get confusing. Why cant we stick to "Reshape" here?

Reply:
@jbobba As I mentioned in first comment to this line `Reshape:v1` is `v1` of `DynReshape`. Current implementation of `get_typeid` (which provides using `switch` over op types) doesn't support it. However, I agree is confusing. Should we create separate task to work on it?

Reply:
Yes, please do.